apiserver/test/ray_cluster_pod_monitor.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: monitoring.coreos.com/v1
```

Comment:
Perhaps this file should be in `./apiserver/deploy/base/`

Reply:
Its an optional one